src/SourceBrowser/src/Directory.Packages.props

```
@@ -12,6 +12,9 @@
     <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.5.0" />
     <PackageVersion Include="Microsoft.Build" Version="17.5.0" />
 
+    <!-- component governance -->
+     <PackageVersion Include="Microsoft.IO.Redist" Version="6.0.1" />

```

```suggestion
    <PackageVersion Include="Microsoft.IO.Redist" Version="6.0.1" />
```

Seems like a minor formotting issue